<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>rosy_visualiser_status_bar</class>
 <widget class="QWidget" name="rosy_visualiser_status_bar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>475</width>
    <height>27</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>27</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>27</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>5</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lblX">
     <property name="text">
      <string>X:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="lblXValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lblY">
     <property name="text">
      <string>Y:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="lblYValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lblZ">
     <property name="text">
      <string>Z:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="lblZValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblRadius">
     <property name="text">
      <string>Radius:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="lblRadiusValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="lblTheta">
     <property name="text">
      <string>Theta:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="lblThetaValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>0.0</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft">
    <widget class="QLabel" name="lblPhi">
     <property name="text">
      <string>Phi:</string>
     </property>
    </widget>
   </item>
   <item alignment="Qt::AlignLeft|Qt::AlignVCenter">
    <widget class="QLabel" name="lblPhiValue">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="frameShape">
      <enum>QFrame::Panel</enum>
     </property>
     <property name="text">
      <string>0.00</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
 <slots>
  <slot>cameraOrientationChanged(float,float,float)</slot>
  <slot>cameraPositionChanged(float,float,float)</slot>
 </slots>
</ui>
